golem_messages/message/tasks.py

```diff
@@ -123,6 +125,11 @@ def deserialize_slot(self, key, value):
         value = super().deserialize_slot(key, value)
         if key == 'compute_task_def':
             value = ComputeTaskDef(value)
+        if key == 'subtask_price':
```

Comment:
+ please add two tests that load `TaskToCompute` with an integer, and a non-integer value in this field... as, sure, a test of the validator is fine, but I was more concerned about the `TaskToCompute` message itself...